app/parse.py

```diff
@@ -8,5 +10,32 @@
     duration: str
 
 
+BASE_URL = "https://mate.academy/"
+
+
+def parse_single_course(soup: BeautifulSoup) -> Course:
+    return Course(
+        name=soup.select_one(".ProfessionCard_title__Zq5ZY").text,
+        short_description=soup.find(
+            "p", {"class": "typography_landingTextMain__Rc8BD mb-32"}
+        ).get_text(strip=True),
+        duration=soup.select_one(
+            ".ProfessionCardTags_regularTag__yTc6K > span"
+        ).text
```

Comment:
Ensure that the CSS selectors used here (e.g., `.ProfessionCard_title__Zq5ZY`, `.typography_landingTextMain__Rc8BD mb-32`, `.ProfessionCardTags_regularTag__yTc6K > span`) match the actual HTML structure of the webpage. If the structure changes, these selectors will need to be updated.